blocks/form/form.js

```diff
@@ -1,74 +1,31 @@
-function createSelect(fd) {
-  const select = document.createElement('select');
-  select.id = fd.Field;
-  if (fd.Placeholder) {
-    const ph = document.createElement('option');
-    ph.textContent = fd.Placeholder;
-    ph.setAttribute('selected', '');
-    ph.setAttribute('disabled', '');
-    select.append(ph);
-  }
-  fd.Options.split(',').forEach((o) => {
-    const option = document.createElement('option');
-    option.textContent = o.trim();
-    option.value = o.trim();
-    select.append(option);
-  });
+function createLabel(fd) {
+  const label = document.createElement('label');
+  label.setAttribute('for', fd.Field);
+  label.textContent = fd.Label;
   if (fd.Mandatory === 'x') {
-    select.setAttribute('required', 'required');
+    label.classList.add('required');
```

Comment:
The old variant is better, because it uses the browser's built-in validation hooks. You can CSS select based on attribute value, so there is no need for this `class`

Reply:
This has been added to support the use cases where someone can add a `*` using css. 

The input elements have require attributed set as well

https://github.com/vdua/nedbank/pull/3/files#diff-14d0faad899dcfcb4e5cd318658f4d58b9e2aa66ebc5912bbe09302339f6a564R37